src/components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -256,18 +270,22 @@
       <value   compset="1850.*_DATM%QIA">1972</value>
       <value   compset="1850.*_DATM%CRU">1920</value>
       <value   compset="1850.*_DATM%GSW">1920</value>
+      <value   compset="1850.*_DATM%NLDAS2">-1</value>  <!-- Unsupported -->
```

Comment:
This NLDAS forcing option is only supported for present-day compsets. For others, I am setting the end year to earlier than the start year, so that preview_namelists will abort with an error like this:

```
ERROR: Stream CLMNLDAS2.Solar starts at year 0, but ends at earlier year -1.
```

I wanted to prevent users from trying to set up (say) a year-1850 or 20th century transient case with these forcing data. However, I didn't feel this was a likely enough event to warrant a lot of work putting in place a more meaningful error message.

Reply:
I think this is good.